libs/net/dal/Migrations/Initial/Up/PostUp/02-DataSourceSchedule.sql

```diff
@@ -0,0 +1,37 @@
+DO $$
+DECLARE DEFAULT_USER_ID UUID := '00000000-0000-0000-0000-000000000000';
+BEGIN
+
+INSERT INTO public.data_source_schedule (
+  "data_source_id"
+  , "schedule_id"
+  , "created_by_id"
+  , "created_by"
+  , "created_on"
+  , "updated_by_id"
+  , "updated_by"
+  , "updated_on"
+  , "version"
+) VALUES (
+  182  -- data_source_id
```

Comment:
It should be a query for the parent by its code. If someone edits this file and adds records it'll be incorrect.